Sprint-2/implement/contains.test.js

```diff
@@ -20,16 +20,33 @@ as the object doesn't contains a key of 'c'
 // Given an empty object
 // When passed to contains
 // Then it should return false
-test.todo("contains on empty object returns false");
+describe("contains",()=>{
+
+    test("contains an empty object returns false", () => {
+    expect(contains({}, 'a')).toEqual(false);
+});
 
 // Given an object with properties
 // When passed to contains with an existing property name
 // Then it should return true
+test("contains with an existing property name", () => {
+    expect(contains({ a: 1, b: 2 }, 'a')).toEqual(true);
+});
+
 
 // Given an object with properties
```

Comment:
I would use `describe` for each Given ... 

`describe("Given an object with properties", () => {.....})`

It is really up to developer how they want to do this but you can always have multiple describe ...